### PR TITLE
Remove simulator hypervisor plugin

### PR DESCRIPTION
**Pull request by miguelaferreira**

### CODE REVIEW

File: agent-simulator/tomcatconf/commands-simulator.properties.in [hunk 1]
```diff
@@ -1,21 +0,0 @@
-# Licensed to the Apache Software Foundation (ASF) under one
```
Comment (miguelaferreira) [inline agent-simulator/tomcatconf/commands-simulator.properties.in]: @borisroman I've removed this file as part of this PR. Can you confirm it relates tot he simulator hypervisor plugin?

**Comment by miguelaferreira** [comment]
Combined build running here: https://beta-jenkins.mcc.schubergphilis.com/job/cosmic/job/0002-tracking-repo-branch-build/130/

**Comment by borisroman** [comment]
@miguelaferreira I couldn't find a single reference to the entire `cosmic-core/agent-simulator/tomcatconf/` directory or the `commands-simulator.properties.in` file.

Removing seems fine by me.

**Comment by miguelaferreira** [comment]
ping @bheuvel

Build is green, see: https://beta-jenkins.mcc.schubergphilis.com/job/cosmic/job/0002-tracking-repo-branch-build/130/

**Pull request merged**